utils/events.js

```diff
@@ -4,13 +4,15 @@ export default [
   {
```

Comment:
add a link to the Date object documentation at the top of this file for future officers